Fix typo in forceatlas2_layout

Bigstool opened the pull request:
As the docstring suggests, the argument should be `distributed_attraction` instead of `distributed_action`.

rossbar reviewed (commented):
For backward-compatibility's sake I think the docstring should be changed instead of the signature

dschult commented:
Thanks for catching and reporting that typo!

I agree with @rossbar that changes to docs are easier/better than changes to keyword arg names in the function signature. And I can't say that the keyword name says much about what is actually happening here with either name. 

If a better name for the keyword is proposed, we could change the kwarg name with a deprecation cycle. But that should be a separate PR. In this PR lets change the docs to match the actual argument used.
Thanks!

Bigstool commented:
Thank you @rossbar and @dschult for the follow-up!

I have reverted the changes to the signature along with the related code and changed the docstring instead.

As I am unfamiliar with deprecation cycles, I will leave the separate PR to the more experienced.

rossbar reviewed (approved):
LGTM, thanks @Bigstool !
